core/src/main/scala/caliban/Configurator.scala

```diff
@@ -0,0 +1,43 @@
+package caliban
+
+import caliban.CalibanError.ValidationError
+import caliban.execution.QueryExecution
+import caliban.validation.Context
+import caliban.validation.Validator.DefaultValidations
+import zio._
+import zio.prelude.EReader
+
+object Configurator {
+  private val validationFiberRef: FiberRef[List[EReader[Context, ValidationError, Unit]]] =
+    Unsafe.unsafe(implicit u => FiberRef.unsafe.make(DefaultValidations))
+
+  private val skipQueryValidationRef: FiberRef[Boolean] =
+    Unsafe.unsafe(implicit u => FiberRef.unsafe.make(false))
+
+  private val enableIntrospectionRef: FiberRef[Boolean] =
+    Unsafe.unsafe(implicit u => FiberRef.unsafe.make(true))
+
+  private val queryExecutionRef: FiberRef[QueryExecution] =
+    Unsafe.unsafe(implicit u => FiberRef.unsafe.make(QueryExecution.Parallel))
```

Comment:
I'm wondering if it would optimize this to make them all part of a single configuration case class? Then during execution phase we only need to examine a single `FiberRef`

Reply:
Done